packages/xo-web/src/xo-app/vm/index.js

```diff
@@ -198,17 +196,15 @@ export default class Vm extends BaseComponent {
                 {vm.power_state === 'Running' && container && (
                   <span>
                     <span> - </span>
-                    <Select
+                    <XoSelect
+                      labelProp='name_label'
                       onChange={this._migrateVm}
-                      options={hosts}
-                      renderer={this._selectOptionRenderer}
                       useLongClick
                       value={container}
+                      xoType='host'
                     >
-                      <Link to={`/${container.type}s/${container.id}`}>
-                        {container.name_label}
-                      </Link>
-                    </Select>
+                      <Host id={container.id} pool={false} link />
+                    </XoSelect>
```

Comment:
Could you actually do this in another tiny PR? :) (technically unrelated)

Reply:
We can't  because we changed `editable/Select` .